luup files/L_RGBController1.lua

```diff
@@ -1037,6 +1037,49 @@ function doColorTransition(lul_device)
 	end
 end
 
+-- MySensor RGB(W)
+RGBDeviceTypes["MYS-RGBW"] = {
+	getParameters = function (lul_device)
+		return {
+			name = "MySensors RGBW",
+			settings = {
+				{ variable = "ArduinoId", name = "Arduino plugin Id", type = "string" },
+				{ variable = "RadioId",   name = "RGB Node altid",    type = "string" }
+			}
+		}
+	end,
+
+	getColorChannelNames = function (lul_device)
+		return {"red", "green", "blue", "warmWhite", "coolWhite"}
```

Comment:
Hello,
I'be pushed a new version. It should work with the MySensors plugin without modification.
You have to copy the action function in the Arduino implementation file.
There's a conflict on "urn:upnp-org:serviceId:SwitchPower1" which is already managed. It seems that the device type has to be checked in this case.

Some test here : 
http://forum.micasaverde.com/index.php/topic,34305.msg252587.html#msg252587